RadixWallet/Features/HomeFeature/Coordinator/Home.swift

```diff
@@ -488,6 +453,21 @@ public struct Home: Sendable, FeatureReducer {
 			}
 		}
 	}
+
+	private func showLinkConnectorIfNeeded() -> Effect<Action> {
+		let purpose: NewConnectionApproval.State.Purpose? = if userDefaults.showRelinkConnectorsAfterProfileRestore {
+			.approveRelinkAfterProfileRestore
+		} else if userDefaults.showRelinkConnectorsAfterUpdate {
+			.approveRelinkAfterUpdate
+		} else {
+			nil
+		}
+		if let purpose {
+			overlayWindowClient.scheduleFullScreenIgnoreAction(.init(root: .relinkConnector(.init(root: .connectionApproval(.init(purpose: purpose))))))
```

Comment:
I was reviewing your other PR when it got merged, but I would rename this to `scheduleFullScreen` since we don't have actions associated to it (as we do on scheduled alerts).